webpack.config.js

```diff
@@ -208,7 +208,7 @@ module.exports = [
                         loader: 'file-loader',
                         options: {
                             outputPath: 'static/assets/',
-                            publicPath: '/static/assets/'
+                            publicPath: 'static/assets/'
```

Comment:
This is definitely going to break scratch-www. I've been looking (without success) for a way to configure webpack that would allow both www and desktop/android to be happy.